modules/UI/videolayout/VideoContainer.js

```diff
@@ -214,6 +214,13 @@ export class VideoContainer extends LargeContainer {
 
         this.$wrapper = $('#largeVideoWrapper');
 
+        // FIXME: the architecture of elements related to the large video and
+        // the naming. The background is not part of largeVideoWrapper because
+        // we are controlling the size of the video trough largeVideoWrapper.
```

Comment:
I don't completely understand this comment. What is the relationship between controlling the size of the video and the background?